internal/topology/variables/cluster_variable_validation_test.go

```diff
@@ -629,7 +630,7 @@ func Test_ValidateClusterVariables(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			g := NewWithT(t)
 
-			errList := validateClusterVariables(tt.values, tt.definitions,
+			errList := validateClusterVariables(context.TODO(), tt.values, nil, tt.definitions,
```

Comment:
Nit, as this is a test, you'll never fix this TODO, better to use Background and forget about it?

Reply:
Or define / use a package-level ctx. Not sure if we already have one in this package

Reply:
Added package-level ctx as suggested - thanks @sbueringer!